spec/presenters/hyrax/iiif_manifest_presenter_spec.rb

```diff
@@ -52,33 +52,8 @@
     let(:solr_doc) { SolrDocument.new(file_set.to_solr) }
     let(:file_set) { create(:file_set, :image) }
 
-    describe '#display_content' do
```

Comment:
Removing this test because #display_content is no longer a method in this repo.  The method is being included from the IIIF AV gem